spec/faraday/options/env_spec.rb

```diff
@@ -34,4 +33,34 @@
     expect(env2[:bar]).to eq(:custom_2)
     expect(subject[:baz]).to be_nil
   end
+
+  describe '#body' do
+    subject { Faraday::Env.from(body: { foo: 'bar' }) }
```

Comment:
Spec Style note - which I just thought of now, but is not critical: When you're going to directly refer to the `subject`, name it.

Suggested standard name: `env` (Provided it's not taken by something else.) So, as definition:

```ruby
subject(:env) { ... }
```
 
That would make it possible to use it like:

```ruby
expect(env.body).to eq({ foo: 'bar' })
```